React-Native/TodoList/src/app/screens/Books/components/List/index.js

```diff
@@ -0,0 +1,32 @@
+import React, { Component } from 'react';
+import { ScrollView, Text, View, Image } from 'react-native';
+import styles from './styles';
+
+export default class List extends Component {
+  bookPlaceholder = 'https://www.rulistings.com/Content/PlaceholderIcons/book_placeholder.png'
```

Comment:
this is OK.. you could have a constant for the placeholder.. but, don't add constants directly inside the class. Add constant outside (after the imports) or a new file with general constans